.github/workflows/rubocop.yml

```diff
@@ -12,8 +12,17 @@ jobs:
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: 2.7
-    - name: Install RuboCop gems
+    - name: Cache gems
+      uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-rubocop-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-rubocop-
+    - name: Install gems
       run: |
-        gem install --no-document rubocop rubocop-performance rubocop-rails
```

Comment:
Wouldn't be easier to specify particular versions in here?

Reply:
The cache time and gem install is down to a combined 2 seconds. It’s far better that Gemfile.lock is the source of truth. Appreciate the suggestion but I think we’re settled on this 😄🙏